views/showcase.pug

```diff
@@ -4,15 +4,15 @@ block content
     h2.text-center.mb-4 Showcase
 
     .row
-        .col-md-6
+        .col-12.col-md-6
             each item in col1
                 include _partials/showcase.pug
-        .col-md-6
+        .col-12.col-md-6
```

Comment:
@XhmikosR what's the diff besides the class name?

Reply:
No difference; one could drop `.col-12` too, but I prefer being explicit.